inst/tests/tests.Rraw

```diff
@@ -15482,10 +15482,10 @@ test(2072.042, fifelse(TRUE,1L,factor(letters[1])),     error="'yes' has differe
 test(2072.043, fifelse(TRUE, list(1:5), list(5:1)), list(1:5))
 test(2072.044, fifelse(as.logical(NA), list(1:5), list(5:1)), list(NULL))
 test(2072.045, fifelse(FALSE, list(1:5), list(5:1)), list(5:1))
-test(2072.046, fifelse(TRUE, data.table(1:5), data.table(5:1)), data.table(1:5))
-test(2072.047, fifelse(FALSE, data.table(1:5), data.table(5:1)), data.table(5:1))
-test(2072.048, fifelse(TRUE, data.frame(1:5), data.frame(5:1)), data.frame(1:5))
-test(2072.049, fifelse(FALSE, data.frame(1:5), data.frame(5:1)), data.frame(5:1))
+test(2072.046, fifelse(TRUE, list(data.table(1:5)), list(data.table(5:1))), list(data.table(1:5)))
```

Comment:
I think it is in-proper use. data.frame should be handled as ordinary list here, thus wrapping into `list` to achieve equality. Consider
```r
fifelse(TRUE, data.table(1:5, 2:6), data.table(5:1, 6:2))
```